src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -762,6 +762,9 @@ class PumpEquipmentDescription:
     esp_parameters = attrib_enum(default=constants.EspParameters.UserDefined)
 
     esp_viscosity_model = attrib_enum(default=constants.PumpViscosityModel.NoModel)
+    thermal_efficiency_model = attrib_enum(
+        default=constants.PumpThermalEfficiencyModel.Constant
+    )
```

Comment:
From
```python
class PumpThermalEfficiencyModel(Enum):
    """
    Defines the ESP thermal efficiency model
```
I believe this should be
```suggestion
    esp_thermal_efficiency_model = attrib_enum(
        default=constants.PumpThermalEfficiencyModel.Constant
    )
```

BTW.
Are non ESP pumps affected by this option?
If they are the enum's docstring should be updated instead.

Reply:
Only the ESP pumps are affected by this option. Therefore, I followed you suggestion and renamed it to `esp_thermal_efficiency_model `.